tsconfig.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
any chance you could sort props by alphabetical order? makes it easier to read objects properties ^w^

Reply:
I'm all for alphabetical everything. Just makes it so much easier to read

Reply:
I'm not familiar with whether this can be done with tsconfig

Reply:
I have no idea either

Reply:
You can do it on tsconfig no problem, done that before